integrity-backend/input-browsertrix-examples/3e11cc57daf3bad8375935cad4878123acc8d769551ff90f1b1bb0dc597-meta-content.json

```diff
@@ -0,0 +1,21 @@
+{
+  "contentMetadata":
+  {
+    "name": "Web archive of @rt_com Twitter",
+    "mime": "application/wacz",
+    "description": "Web archive of @rt_com Twitter on 2022-03-29",
```

Comment:
This may be difficult to generate, especially if you have 100 links in a CSV file.

Reply:
I think this can be very flexible. As descriptive as we can manage. For example it can say:

>"Web archive curated from X Lab with identifier 3064 on 2022-03-29".